test/relative-time.js

```diff
@@ -159,4 +159,22 @@ suite('relative-time', function() {
     }
     assert.equal(root.children[0].textContent, 'now')
   })
+
+  test('rewrites given lang attribute', function() {
+    const now = new Date(Date.now() - 3 * 60 * 60 * 24 * 1000).toISOString()
+    const time = document.createElement('relative-time')
+    time.setAttribute('datetime', now)
+    time.setAttribute('lang', 'es')
+    assert.equal(time.getFormattedDate(), 'hace 3 días')
+  })
+
+  test('rewrites given parent lang attribute', function() {
+    const container = document.createElement('span')
+    container.setAttribute('lang', 'es')
+    const now = new Date(Date.now() - 3 * 60 * 60 * 24 * 1000).toISOString()
+    const time = document.createElement('relative-time')
+    time.setAttribute('datetime', now)
+    container.appendChild(time)
```

Comment:
At this time, the contents of the `relative-time` element is already rendered, and re-parenting it (so it inherits the parent's `lang` attribute) won't change the already rendered contents, I think. Perhaps for the sake of the test, you could do `setAttribute('datetime', ...)` after it has been added to the correct parent?